### PR TITLE
Added the timeline link of GSoC 2020 in Readme.md

### PR DESCRIPTION
**Pull request by ayan-biswas0412**
I have edited the Readme.md with the new timelines.
And also added the guideline for the participants so that they update the new timeline in the schedule of Deliverables in the Project
@alranel Another small contribution against [#99](https://github.com/arduino/summer-of-code/issues/99)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,26 @@ Arduino is fully committed to creating open-source hardware and software. The Ar
...
+
+This information is Also available at GSoC 2020 Portal [Click Here](https://summerofcode.withgoogle.com/how-it-works/#timeline)
+
+**Don't Forget to update your Schedule of Deliverables(if any) in your project proposal** 
+
+
+
+* Open an issue in this [issue tracker](https://github.com/arduino/summer-of-code/issues) (only **one issue per student**): you can use it to ask questions and communicate with us.
+
+The issue tracker of this repository is the preferred way to communicate with us. Should you have private inquiries, send an e-mail to [summer-of-code@arduino.cc](mailto:summer-of-code@arduino.cc).
```
Comment (per1234) [inline README.md]:
```suggestion



* Open an issue in this [issue tracker](https://github.com/arduino/summer-of-code/issues) (only **one issue per student**): you can use it to ask questions and communicate with us.

The issue tracker of this repository is the preferred way to communicate with us. Should you have private inquiries, send an e-mail to [summer-of-code@arduino.cc](mailto:summer-of-code@arduino.cc).
```
This is duplicate text.

**Review by per1234 — CHANGES_REQUESTED** [review]
I don't think it's a good idea to specify the timeline in this repository. The reason is that is Google changes the information again later we would be providing incorrect information. If we want to document the timeline in the readme, it's better to just provide a link to the official information on Google's website.

**Comment by ayan-biswas0412** [comment]
@per1234  Review request Rectified according to your review comment .

File: README.md (hunk 1 above)
Comment (ayan-biswas0412) [reply README.md]: Done that successfully

**Pull request closed without merging**